app/models/amusement_park.rb

```diff
@@ -1,3 +1,4 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanics, through: :rides
```

Comment:
Great job adding this association. In here is where you would want to create a method to get unique mechanics so that you don't have it in your `amusement_park` controller. So something like:
```ruby
def unique_mechanics
  mechanics.distinct
end
```
Then you will call this method in your `amusement_parks/show.html.erb`.